docs/source/security/cors.mdx

```diff
@@ -84,108 +82,135 @@ If you create a public API or an API to embed in websites you don't control your
+Using the `cors` package directly, we can configure the `Access-Control-Allow-Origin` header using the [`origin` option](https://github.com/expressjs/cors#configuration-options). The example above enables CORS requests from `https://www.your-app.example`, along with `https://studio.apollographql.com`.
 
-If you're using another integration of Apollo Server, you can add the `cors` configuration option to your framework-specific middleware function to change your server's CORS behavior. To learn more about the CORS defaults and options for your integration of Apollo Server, see [CORS configuration options](../api/apollo-server/#cors-1).
+> Note: if you plan to use [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/explorer/) as a GraphQL web IDE, you should include `https://studio.apollographql.com` in your list of valid origins. However, if you plan to embed the [Explorer](https://www.apollographql.com/docs/studio/explorer/embed-explorer/) or use [Apollo Sandbox](https://www.apollographql.com/docs/studio/explorer/sandbox), you *don't* need to specify Studio's URL in your CORS origins because requests will go through the page embedding Studio.
 
-Below is an example of setting up CORS with the [`apollo-server-express`](../integrations/middleware#swapping-out-apollo-server) package:
+If you're using another integration of Apollo Server, you can add and configure CORS for your server using your framework's standard functionality. 
 
-```js
-// ... set up the server
-await server.start();
+You can also choose to remove CORS middleware entirely to disable cross-origin requests. If you are using `expressMiddleware`, you can do this by passing `false` to the `cors` function:
 
-const corsOptions = {
-  origin: ["https://www.your-app.example", "https://studio.apollographql.com"]
-};
+<MultiCodeBlock>
 
-server.applyMiddleware({
-  app,
-  cors: corsOptions,
-  path: "/graphql",
-});
+```ts
+app.use(
+  '/graphql',
+  // highlight-start
+  cors<cors.CorsRequest>(false),
+  // highlight-end
+  json(),
+  expressMiddleware(server),
+);
 ```
 
-You can also [remove CORS middleware entirely](../api/apollo-server#cors) to disable cross-origin requests. In the batteries-included `apollo-server` and `apollo-server-express`, you do this by passing `cors: false`. This is [recommended for subgraphs in a federated graph](/federation/subgraphs/#securing-your-subgraphs).
+</MultiCodeBlock>
+
+This is [recommended for subgraphs in a federated graph](/federation/subgraphs/#securing-your-subgraphs).
 
 ### Passing credentials with CORS
```

Comment:
This section should be explicit that you just can't do it with startStandaloneServer.

Reply:
Love it! Adding a callout at the beginning of the "Passing credentials with CORS" section:

> ⚠️ The [`startStandaloneServer`](../migration#migrate-from-apollo-server) function's CORS behavior is _unalterable_. To pass credentials via cookies, you must first swap to another Apollo Server integration.